content/mailing_lists.html

```diff
@@ -184,6 +184,13 @@ <h2 id="security">Security</h2>
 <p>In order to post to the list, it is <strong>NOT</strong> necessary to first subscribe
 to it.</p>
 <p>For information on published vulnerabilities please see our <a href="cve_list.html">CVE list</a>.</p>
+<p>This mailing list is only for discussing security vulnerabilities in hadoop &lsquo;source&rsquo; code, <strong>NOT</strong> security vulnerabilities coming via thirdparty libraries. For security issues related to thirdparty libraries use the dev/user mailing lists.</p>
```

Comment:
When talking about dependencies, I like to say 'advisories for dependencies' rather than 'vulnerabilities in dependencies', to make it clearer that often the dependency is used in a way that doesn't lead to a vulnerability in Hadoop. Howver, when it turns out the upstream problem does have impact, that might be something to discuss privately... perhaps something like:

```suggestion
<p>This mailing list is only for discussing security vulnerabilities in hadoop &lsquo;source&rsquo; code, <strong>NOT</strong> security advisories for thirdparty libraries. For security issues related to thirdparty libraries use the dev/user mailing lists. However, when after analysis it turns out the advisory impacts Hadoop, that should be discussed on the security list.</p>
```